### DISCUSSION TITLE
Unable to generate new title

### DISCUSSION THREAD

**Discussion by GomoXU** [danny-avila/LibreChat#3493] [Troubleshooting]
### What happened?

When you create a new chat and paste an image into it, the chat will not generate a title, but will only show up as New Chat.

### Steps to Reproduce

1.create a new chat 
2. paste an image into it, 
3. the chat will not generate a title, but will only show up as New Chat.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by berry-13** [comment]
which endpoint?

**Comment by GomoXU** [comment]
> which endpoint?

azure openai api
and title model is gpt-35-turbo

**Comment by berry-13** [comment]
I can't test it with Azure OpenAI. OpenAI seems to be working correctly, maybe it's an Azure issue, so I'll leave it open

**Comment by GomoXU** [comment]
> I can't test it with Azure OpenAI. OpenAI seems to be working correctly, maybe it's an Azure issue, so I'll leave it open

I changed the titleModel to “gpt-4o” and it  works now, I think the reason I can't generate the title is because I chose gpt-35-turbo, maybe can fix this by try to generate the title using the input text instead of using an image.
